wiki/Introduction-to-Creating-a-Job.md

```diff
@@ -0,0 +1,1818 @@
+          type: INT
+          range: "{{Param.FrameStart}}-{{Param.FrameEnd}}"
+    script:
+      actions:
+        onRun:
+          command: "{{Task.File.Render}}"
+          args:
+            - "{{Param.SceneFile}}"
+            - "{{Param.FramesDirectory}}"
+            - "{{Task.Param.Frame}}"
+          timeout: 60
+      embeddedFiles:
+        - name: Render
+          type: TEXT
+          filename: render.sh
+          runnable: true
+          data: |
+            #!/bin/bash
+
+            # Return an error code if any command in the script fails.
+            set -eou pipefail
+
+            TMPFILE=$(mktemp)
+            cat > "$TMPFILE" << EOF
+            import bpy
+            for s in bpy.data.scenes:
+              s.render.resolution_x = 480
+              s.render.resolution_y = 270
+            bpy.context.scene.cycles.samples = 100
+            EOF
+
+            SCENE="$1"
+            OUTDIR="$2"
+            FRAME="$3"
+
+            blender --background "$SCENE" \
+              --python "$TMPFILE" \
+              --render-output "$OUTDIR"/frame-### \
+              --render-format PNG --use-extension 1 \
+              --render-frame "${FRAME}"
+
+            rm -f $TMPFILE
+
+  - name: EncodeVideo
+    dependencies:
+      - dependsOn: BlenderRender
+    script:
+      actions:
+        onRun:
+          command: "{{Task.File.Encode}}"
+          args:
+            - "{{Param.FramesDirectory}}"
+            - "{{Param.AnimationFile}}"
+            - "{{Param.FrameStart}}"
+          timeout: 60
+      embeddedFiles:
+        - name: Encode
+          type: TEXT
+          runnable: true
+          filename: encode.sh
+          data: |
+            #!/bin/bash
+
+            set -eou pipefail
+
+            INPUT_DIR="$1"
+            OUTPUT_FILENAME="$2"
+            START_FRAME="$3"
+
+            ffmpeg -y -r 10 -start_number "$START_FRAME" -i "$INPUT_DIR"/frame-%03d.png -pix_fmt yuv420p \
+                -vf "scale=in_color_matrix=bt709:out_color_matrix=bt709" \
+                -frames:v 300 -c:v libx264 -preset fast \
+                -color_range tv -colorspace bt709 -color_primaries bt709 -color_trc iec61966-2-1 \
+                -movflags faststart "$OUTPUT_FILENAME"
+
+```
+
+#### 2.2.4.2. Multiple Frames Per Task
+
+Some rendering applications may take a significant amount of time to startup and load the data that it needs to work. In this case,
+you may wish to render multiple contiguous frames of the animation in each Task. There are two ways that you can accomplish this
+with Open Job Description:
+
+1. Define Task parameters for the start and end of the range of frames to render in each Task and leverage a
+   [combination expression](2023-09-Template-Schemas#343-combinationexpr) to combine them; or
+2. Define Task parameter as a `STRING` type and hardcode the range of frames that we want to render in each Task.
+
+We'll go through both options.
+
+##### 2.2.4.2.1. With a Combination Expression
```

Comment:
This section needs the feature request https://github.com/OpenJobDescription/openjd-model-for-python/issues/131 to be implemented before it can be published.